css/index.css

```diff
@@ -4,372 +4,484 @@
+q,
+s,
+samp,
+small,
+strike,
+strong,
+sub,
+sup,
+tt,
+var,
+b,
+u,
+i,
+center,
+dl,
+dt,
+dd,
+ol,
+ul,
+li,
+fieldset,
+form,
+label,
+legend,
+table,
+caption,
+tbody,
+tfoot,
+thead,
+tr,
+th,
+td,
+article,
+aside,
+canvas,
+details,
+embed,
+figure,
+figcaption,
+footer,
+header,
+hgroup,
+menu,
+nav,
+output,
+ruby,
+section,
+summary,
+time,
+mark,
+audio,
+video {
+  margin: 0;
+  padding: 0;
+  border: 0;
+  font-size: 100%;
+  font: inherit;
+  vertical-align: baseline;
 }
 /* HTML5 display-role reset for older browsers */
-article, aside, details, figcaption, figure, 
-footer, header, hgroup, menu, nav, section {
-	display: block;
+article,
+aside,
+details,
+figcaption,
+figure,
+footer,
+header,
+hgroup,
+menu,
+nav,
+section {
+  display: block;
 }
 body {
-	line-height: 1;
+  line-height: 1;
 }
-ol, ul {
-	list-style: none;
+ol,
+ul {
+  list-style: none;
 }
-blockquote, q {
-	quotes: none;
+blockquote,
+q {
+  quotes: none;
 }
-blockquote:before, blockquote:after,
-q:before, q:after {
-	content: '';
-	content: none;
+blockquote:before,
+blockquote:after,
+q:before,
+q:after {
+  content: "";
+  content: none;
 }
 table {
-	border-collapse: collapse;
-	border-spacing: 0;
+  border-collapse: collapse;
+  border-spacing: 0;
 }
 
 /* Set every element's box-sizing to border-box */
 * {
-    box-sizing: border-box;
+  box-sizing: border-box;
 }
 
-html, body {
-    height: 100%;
-    font-family: 'Titillium Web', sans-serif;
+html,
+body {
+  height: 100%;
+  font-family: "Titillium Web", sans-serif;
 }
 
-h1, h2, h3, h4, h5 {
-    font-family: 'Bangers', cursive;
-    letter-spacing: 1px;
-    margin-bottom: 15px;
+h1,
+h2,
+h3,
+h4,
+h5 {
+  font-family: "Bangers", cursive;
+  letter-spacing: 1px;
+  margin-bottom: 15px;
 }
 
 /* Copy and paste your work from yesterday here and start to refactor into flexbox */
 .container {
-	width: 1080px;
-	
-	/* border: 10px dashed black; */
-	margin: 0 auto;
+  max-width: 1080px;
+
+  /* border: 10px dashed black; */
+  margin: 0 auto;
 }
-@media (max-width: 800px) {.container {
-    max-width: 700px;}
+@media (max-width: 800px) {
+  .container {
+    max-width: 700px;
+  }
 }
 
-@media (max-width: 500px) {.container {
+@media (max-width: 500px) {
+  .container {
     max-width: 400px;
-}
+  }
 }
 
 header {
-	/* border: 5px dashed orange; */
-	padding: 20px 20px 20px;
+  /* border: 5px dashed orange; */
+  padding: 20px 20px 20px;
   display: flex;
   align-items: center;
-
 }
 
-@media (max-width: 500px) {header {
+@media (max-width: 500px) {
+  header {
     flex-flow: column;
     justify-content: center;
     max-width: 400px;
     flex-direction: column-reverse;
-}}
+  }
+}
 
-@media (max-width: 800px) {header {
-    max-width: 700px;
+@media (max-width: 800px) {
+  header {
+    /* max-width: 700px; */
     flex-flow: column;
     justify-content: center;
-       flex-direction: column-reverse;
-}}
+    flex-direction: column-reverse;
+  }
+}
 
 .logo {
-	margin: 5%;
+  margin: 5%;
```

Comment:
Try cutting down on the top margin so that your page starts near the top of the viewport.